jekyll/_cci2/docker-layer-caching.md

```diff
@@ -11,7 +11,7 @@ order: 70
 
 {% include beta-premium-feature.html feature='Docker Layer Caching' %}
 
-This document describes how to enable Docker Layer Caching (DLC) which is useful when you are _building_ Docker images during your job or Workflow. DLC does **not** speed up downloading of Docker images used to _run_ your jobs. 
+This document describes how to enable Docker Layer Caching (DLC), which is useful when you are _building_ Docker images during your job or Workflow. DLC does **not** speed up downloading of the Docker images used to _run_ your jobs (i.e., any images downloaded in a job's "Spin up Environment" step).
```

Comment:
Suggest changing this to avoid parenthetical, possessive 'job's' and use of i.e.
(i.e., ... )-> That is, Docker images that are used to run jobs appear in the Spin up Environment step for a job and are *not* cached by DLC.

Reply:
sorry, so, you want me to push another commit & make your suggested changes ? @michelle-luna

Reply:
oh no, I will do it, just wanted to check with you first for some reason (Mondays!?)

Reply:
ok sure, feel free to push new changes & let me review !! thankssssssss @michelle-luna